README.md

```diff
@@ -118,6 +118,7 @@ These settings are specific to VSCodeVim.
 | vim.disableExtension             | Disable VSCodeVim extension. This setting can also be toggled using `toggleVim` command in the Command Palette                                                                                                                                                                                                                                                                                                                      | Boolean | false                                 |
 | vim.handleKeys                   | Delegate configured keys to be handled by VS Code instead of by the VSCodeVim extension. Any key in `keybindings` section of the [package.json](https://github.com/VSCodeVim/Vim/blob/master/package.json) that has a `vim.use<C-...>` in the `when` argument can be delegated back to VS Code by setting `"<C-...>": false`. Example: to use `ctrl+f` for find (native VS Code behaviour): `"vim.handleKeys": { "<C-f>": false }`. | String  | `"<C-d>": true`                       |
 | vim.overrideCopy                 | Override VS Code's copy command with our own, which works correctly with VSCodeVim. If cmd-c/ctrl-c is giving you issues, set this to false and complain [here](https://github.com/Microsoft/vscode/issues/217).                                                                                                                                                                                                                    | Boolean | false                                 |
+| vim.useSystemClipboard           | Replace the unnamed register with the system clipboard. Pressing d/y/p alone in normal mode will use VS Code's native cut/copy/paste functionality instead of a Vim register.                                                                                                                                                                                                                                                       | Boolean | false                                 |
```

Comment:
The second bit isn't quite right - we don't use VSCode's native cut/copy/paste, just the system clipboard

Reply:
Would it be more accurate to say that it behaves identically to VS Code's cut/copy/paste, or is there a difference I'm missing?

(I interpreted the calls to `vscode.env.clipboard` in https://github.com/VSCodeVim/Vim/blob/master/src/util/clipboard.ts to be equivalent to the default user-facing controls.)

Reply:
Should I open a new PR to fix the merge conflicts cleanly? with wording like
> Replace the unnamed register with the system clipboard. Pressing d/y/p alone in normal mode will use the system clipboard instead of a Vim register.